std_msgs/Header header
float32 thrust
float32 M1_hat
float32 M2_hat
float32 M3_hat
geometry_msgs/Vector3 position_error
geometry_msgs/Vector3 position_error_integral
geometry_msgs/Vector3 velocity_error
geometry_msgs/Vector3 angle_error
geometry_msgs/Vector3 angle_rate_error
geometry_msgs/Vector3 acceleration
geometry_msgs/Vector3 angular_acceleration
geometry_msgs/Vector3 moments
geometry_msgs/Vector3 Kp1_hat
geometry_msgs/Vector3 Kp2_hat
geometry_msgs/Vector3 Kp3_hat
geometry_msgs/Vector3 Kq1_hat_0
geometry_msgs/Vector3 Kq2_hat_0
geometry_msgs/Vector3 Kq3_hat_0
geometry_msgs/Vector3 Kq1_hat_1
geometry_msgs/Vector3 Kq2_hat_1
geometry_msgs/Vector3 Kq3_hat_1
geometry_msgs/Vector3 Kq1_hat_2
geometry_msgs/Vector3 Kq2_hat_2
geometry_msgs/Vector3 Kq3_hat_2
geometry_msgs/Vector3 sp
geometry_msgs/Vector3 sq
geometry_msgs/Vector3 zeta_p
geometry_msgs/Vector3 zeta_q
geometry_msgs/Vector3 delTau_p
geometry_msgs/Vector3 delTau_q
geometry_msgs/Vector3 p1_rho
geometry_msgs/Vector3 p2_rho
geometry_msgs/Vector3 p3_rho
geometry_msgs/Vector3 q1_rho_0
geometry_msgs/Vector3 q2_rho_0
geometry_msgs/Vector3 q3_rho_0
geometry_msgs/Vector3 q1_rho_1
geometry_msgs/Vector3 q2_rho_1
geometry_msgs/Vector3 q3_rho_1
geometry_msgs/Vector3 q1_rho_2
geometry_msgs/Vector3 q2_rho_2
geometry_msgs/Vector3 q3_rho_2
